lib/webrtc/call.js

```diff
@@ -74,6 +74,12 @@ MatrixCall.ERR_LOCAL_OFFER_FAILED = "local_offer_failed";
  */
 MatrixCall.ERR_NO_USER_MEDIA = "no_user_media";
 
+/**
+ * Lookup from opaque queue ID to a promise for media element operations that
+ * need to be serialised into a given queue
+ */
+MatrixCall.mediaPromises = {};
```

Comment:
Why are you attaching this to the prototype rather than being an instance member of the `MatrixCall` like the other fields are?